### PR TITLE
Feature/update kass configs

### PR DESCRIPTION
**Pull request by pslocum**
These changes include updates to the Kassiopeia config files for alignment with https://github.com/project8/hexbug/pull/10 .  Field solutions are moved from the Kassiopeia to the KEMField namespace.

### CODE REVIEW

File: Config/LocustKass_FreeSpace_Template.xml.in [hunk 1]
```diff
@@ -1,66 +1,29 @@
-
 <!-- parameters -->
-
-<external_define name="fieldX" value="0.0"/>
-<external_define name="fieldY" value="0.0"/>
-
- <external_define name="generator" value="gen_uniform"/>
+<external_define name="simulation" value="1"/>
```
Comment (renereimann) [inline Config/LocustKass_FreeSpace_Template.xml.in:2]: This line can be deleted. The variable is never used.

**Review by renereimann — CHANGES_REQUESTED** [review]
### Requested Changes

**Config/LocustKass_Waveguide_Template.xml.in** 
line 6 and 7 need to be deleted. The parameters are defined in WaveguideGeometry and would be duplicated. It would not be clear which one is the used one and users changing the value may get the same result because the other parameter definition is overwriting the change.
**Config/JustKassFieldMap.xml.in** 
line 6 and 7 need to be deleted. Same argument as above

### Notes (side modifications not specific to the pull request but in general okay)
- in Config/JustKassFieldMap.xml.in also vtk write / window is commented out
- in Config/JustKassFieldMap.xml.in also geo_surface is commented out
- in Config/LocustKass_FreeSpace_Template.xml.in also krypton generator is commented out
- in Config/LocustKass_FreeSpace_Template.xml.in also min_energy terminator is removed
- in Config/LocustKass_FreeSpace_Template.xml.in also root_writer for component_step_world is added

### Not clear to me:
- in Config/LocustKass_FreeSpace_Template.xml.in in line 341 the `<cycl_rad_extr name="rad_extr" P8Phase="0" />` was changed to 3. I am not sure about what that effect this has and if that is okay. I guess @pslocum can judge that the best and should add a comment.

File: Config/LocustKass_FreeSpace_Template.xml.in (hunk 1 above)
Comment (pslocum) [reply Config/LocustKass_FreeSpace_Template.xml.in]: The line is now deleted.  Thanks.

**Comment by pslocum** [comment]
Thanks for the review.  Here are the updated changes and comments:

* The duplicate definitions of fieldX=0 and fieldY=0 are now removed.
* vtk writing is commented out as we do not yet have kassiopeia with vtk in the container.  It is still being upgraded.
* geo_surface waveguide termination is commented out in the field map xml as it is inert.
* The krypton generator and min_energy termination are left commented out as they are also unused in the tutorial examples.  In the meantime they are left in the xml for optional inspection by a user.
* root_writer for component_step_world is left in place.  It was imported from hexbug and could be useful for a new user to see how to save trajectory information.
* p8phase=3 is now reverted to p8phase=0 after being imported from hexbug.  This allows running the simulation without p8-specific calculations.

**Review by renereimann — APPROVED** [review]
All fine from my side

**Pull request merged**